codespell_lib/data/dictionary.txt

```diff
@@ -14072,6 +14077,7 @@ expierience->experience
 expieriences->experiences
 expilicitely->explicitly
 expireitme->expiretime
+expiried->expired
```

Comment:
If you do not mind add these
```text
expirie->expire
expiriation->expiration
expiriy->expiry
```
* https://github.com/search?q=expiriation&type=code
* https://github.com/search?q=expirie&type=code
* https://github.com/search?q=expiriy&type=code